### ISSUE TITLE
commands which are meant to connect to "process-compose" via REST API fail silently if server is not accessible or returns unexpected result

### ISSUE THREAD

**Issue by adrian-gierakowski** [F1bonacc1/process-compose#69]
## Defect

running commands like:

```sh
process-compose process list
```

doesn't exit with non-zero status if server is not accessible, or returns unexpected result 

#### Version of `process-compose`:

```
Process Compose
Version:        v0.51.4
Commit:         e3cc52e
```

#### OS environment:

nixos

#### Steps or code to reproduce the issue:

make sure there is no `process-compose` running on default port and run

```sh
process-compose process list
```

#### Expected result:

exits with non-zero status, and prints an error that it could not connect to server on given port 

if some sever is accessible but returns unexpected result, this should also result non-zero status and a message that connection was established but requests failed, suggesting that some process other than process-compose might be running on given host\port

#### Actual result:

exists silently with status 0

**Comment by F1bonacc1** [comment]
Good catch.
Will be fixed in the next release.

**Comment by F1bonacc1** [comment]
Should be fixed in v0.60.0
Feel free to reopen if it doesn't work as expected.
